iothub_client/src/iothub_client_edge.c

```diff
@@ -368,6 +369,8 @@ static STRING_HANDLE createRelativePath(const char* deviceId, const char* module
     return result;
 }
 
+static const char* ENVIRONMENT_VAR_EDGEHUB_CACERTIFICATEFILE = "EdgeModuleCACertificateFile";
```

Comment:
Nitpick: Can you please move this up with the other const char*, just so they're all together

Reply:
Fixed.